setup.py

```diff
@@ -16,6 +16,16 @@
 
 import kalite
 
+# Since pip 7.0.1, bdist_wheel has started to be called automatically when
```

Comment:
And somehow `wheel` is now a dependency of `pip`, at least I had it automatically installed on my vanilla Ubuntu after doing `pip install pip --upgrade`.